pkg/sriov/list.go

```diff
@@ -0,0 +1,80 @@
+package sriov
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListNetworkNodeState returns NetworkNodeStates inventory in the given namespace.
+func ListNetworkNodeState(apiClient *clients.Settings,
```

Comment:
Please fix according to convention:
https://github.com/openshift-kni/eco-goinfra#functions-format

Reply:
fixed